apps/C/breakout.c

```diff
@@ -0,0 +1,222 @@
+#include "../defines/rvc_defines.h"
+#include "../defines/graphic_screen.h"
+
+/* This function print a char note on the screen in (raw,col) position */
+void draw_char(char note, int raw, int col)
```

Comment:
I know I keep bringing this up...
Why can't this function (and all the graphic functions) be in a file that is "included"?
Similar to this:
![image](https://user-images.githubusercontent.com/81047407/184935719-543b6d23-0cc9-4dec-b0be-fd3ed16019bd.png)